maven-core/src/main/java/org/apache/maven/project/artifact/ActiveProjectArtifact.java

```diff
@@ -65,6 +66,12 @@ public File getFile()
         return project.getArtifact().getFile();
     }
 
+    /** {@inheritDoc} */
+    public Path getPath()
+    {
+        return this.getFile() == null ? null : this.getFile().toPath();
+    }
+
```

Comment:
did you override default implementation from interface?

Reply:
Yes, implicitly, as this class extends `DefaultArtifact`.